frontend/src/styles/AppHeader.css

```diff
@@ -1,5 +1,7 @@
 .userIcon {
     height: 20px;
     width: 20px;
-    border: 1px solid var(--light-font);
+    top: 98px;
+    left: 342px;
+    color: rgba(244, 245, 248, 0.62);
 }
```

Comment:
Whole AppHeader.css can be removed, I don't think we need any custom styling for it anymore.

Reply:
we have different color rgba(244, 245, 248, 0.62) for user icon  so can i only add this color in css?

Reply:
Right, true, we can actually ignore this with this feature, I'll consider the same in the styling refactoring branch!